### PR TITLE
Scripts to automate sending requests messages in LinkedIn

### LINKED ISSUE #278: Send LinkedIn Connection Requests
# Description

Create a Script to send LinkedIn Connection Requests in bulk from this [endpoint](https://www.linkedin.com/mynetwork/) with an argument as a Custom Connection Message. You can specify how many people you may want to send the Connection Request. 

## Language

- [X] Bash
- [X] Go
- [X] Javascript
- [X] Python

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by Namyalg**
# Description

This script can be used to send multiple connection requests on LinkedIn from the mentioned [endpoint](https://www.linkedin.com/mynetwork/) with a custom message

Fixes #278 

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Automate-LinkedIN/automate_linkedin.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#Import and dependencies
+from selenium import webdriver
+import time
+from selenium.webdriver.common.keys import Keys
+ten = 10
```
Comment (srimani-programmer) [inline Automate-LinkedIN/automate_linkedin.py:5]:
```suggestion
please use proper naming convention for variables.

```

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
Use the Proper naming conventions and also follow the proper formatting guidelines to format the code. Please refractor your code once again and make appropriate changes to it.

**Comment by Namyalg** [comment]
I will make the necessary changes.

**Comment by Namyalg** [comment]
I have made the changes, I would like to know if there is anything else I have to change

File: Automate-LinkedIN/automate_linkedin.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                    driver.find_element_by_xpath("/html/body/div[4]/div/div/div[2]/div[1]/textarea").send_keys(message)
+                    time.sleep(time_wait_two)
+                    driver.find_element_by_xpath("/html/body/div[4]/div/div/div[3]/button[2]").click()
+                    time.sleep(time_wait_two)
+                except:
+                    print("Cannot connect")
+        except:
+            print("Cannot connect")
+        print("Do you want to make another connection? ")
+        done = input(input("Enter 1 to continue, 0 to stop "))
```
Comment (srimani-programmer) [inline Automate-LinkedIN/automate_linkedin.py:73]:
```suggestion
      #  please add the proper variable convention here as well.

```

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
This script is getting collapsed if the user has set the default chrome driver path. please add the exception block to handle those exceptions.

**Comment by Namyalg** [comment]
Okay, I will check

**Comment by Namyalg** [comment]
I have made the changes

**Pull request merged**